src/neural_sde.jl

```diff
@@ -0,0 +1,88 @@
+using Flux, DiffEqFlux, StochasticDiffEq,LinearAlgebra
+d = 3
+hls = 10 + d
+# _u0 = Flux.data(u0(x0)[1])
+
+datasize = 5
+tspan = (0.0f0,1.0f0)
+t = range(tspan[1],tspan[2],length=datasize)
+
+f(X, u, _σᵀ∇u, p, t) = 0.5 * (u .- sum(X.*_σᵀ∇u))
+μ(X,p,t) = 1.0
+σ(X,p,t) = 1.0
+
+u0 = Flux.Chain(Dense(d,hls,relu),
+                Dense(hls,hls,relu),
+                Dense(hls,1))
+
+σᵀ∇u = Flux.Chain(Dense(d,hls,relu),
+                  Dense(hls,hls,relu),
+                  Dense(hls,d))
+g(X) = sum(X.^2)
+
+function trueODEfunc(dh,h,p,t)
+    u =  h[end]
+    X =  h[1:end-1]
+    _σᵀ∇u = σᵀ∇u(X) #TODO tracked σᵀ∇u
+    dh[end] = Flux.data(f(X, u, _σᵀ∇u, p, t))
+    dh[1:end-1] .= μ(X,p,t)
+    dh
+end
+
+function true_noise_func(dh,h,p,t)
+    X = h[1:end-1]
+    _σᵀ∇u = Flux.data(σᵀ∇u(X))
+    dh[end] = sum(_σᵀ∇u) # TODO _σᵀ∇u*dW nondioganal
```

Comment:
yeah this isn't correct, but I think that works just for this case of sigma = 1?